Vishnu_45/html/Assignment3/ass2_mean.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
need proper code indentation
add new line
change all with at the end of css property ends